### PR TITLE
Adding Filter

### PR DESCRIPTION
**Pull request by pjhale**
Default returns the normal data set... the other two return only  references that have a specific workflow tag.

### CODE REVIEW

**Comment by pjhale** [comment]
Added in download button.  Moved the button out of the pagination element into its own element.  Made into a bootstrap button. @sweng66

**Comment by pjhale** [comment]
This works... however the tag "_missing_files" is hardcoded into the UI here.  If you directly query the API you get the correct file name.

**Review by sweng66 — APPROVED** [review]
Looks great! Thanks @pjhale

File: src/components/Tracker.js [hunk 1]
```diff
@@ -1,18 +1,64 @@
...
 import LoadingOverlay from "./LoadingOverlay";
 
+const DownloadButton = (mod) => {
+  const [downloading, setDownloading] = useState(false);
+  const orderBy = useSelector(state => state.tracker.orderBy);
+  const trackerFilter = useSelector(state => state.tracker.trackerFilter);
+
+  const handleDownload = () => {
+    setDownloading(true);
+    let apiUrl = `${process.env.REACT_APP_RESTAPI}/reference/download_tracker_table/${mod.mod}?order_by=${orderBy}&filter=${trackerFilter}`;
```
Comment (azurebrd) [inline src/components/Tracker.js:25]: I thought mod was accessLevel which is a string, but if mod.mod works, then cool.

Comment (azurebrd) [reply src/components/Tracker.js:25]: Cool, thanks @pjhale

**Pull request merged**